test/accounts_cycles_test.go

```diff
@@ -21,6 +21,7 @@ import (
 	"time"
 
 	"memphis/server"
+
```

Comment:
you should delete the changes in this file I guess you replaced all strings partitionNumber